server/src/main/java/org/hyperskill/community/flashcards/card/model/Card.java

```diff
@@ -31,9 +30,8 @@ public abstract sealed class Card permits QuestionAndAnswerCard, SingleChoiceQui
 
     public Card() { }
 
```

Comment:
what is the public constructor for? does mongo need it?

Reply:
Removed unused constructors, they are not required.